Cargo.toml

```diff
@@ -40,7 +40,7 @@ debug = 2
 # chalk-ir = { path = "../chalk/chalk-ir" }
 # chalk-recursive = { path = "../chalk/chalk-recursive" }
 # chalk-derive = { path = "../chalk/chalk-derive" }
-# line-index = { path = "lib/line-index" }
+line-index = { path = "lib/line-index" }
```

Comment:
Please split this PR up into two, one changing the lib/line-index crate, that we then merge and have a patch release done by. Then we can rebase this PR on top to use the new release